### ISSUE TITLE
CFP: Multicast IPSec Support

### ISSUE THREAD

**Issue by ldelossa** [cilium/cilium#29471]
## Multicast IPSec Support
        
Implement IPSec support for multicast traffic. 

Dependent on #29469

**Comment by ldelossa** [comment]
Completed and merged into `main-ce` 

UBS has a custom made image based on v1.15-ce.

Once v1.16-ce is released that branch can be destroyed and they can be moved to the traditional release cadence.
